components/tyk-dev-portal/templates/_helpers.tpl

```diff
@@ -56,4 +56,31 @@ app.kubernetes.io/instance: {{ .Release.Name }}
     {{- else }}
         {{- tpl (.value | toYaml) .context }}
     {{- end }}
+{{- end -}}
+
+{{/*
+    It lists all services in the release namespace and find a service
+    for Tyk Dashboard with its label.
+*/}}
+{{- define "tyk-dev-portal.dashboardSvcName" -}}
+   {{- $services := (lookup "v1" "Service" .Release.Namespace "") -}}
+   {{- if $services -}}
+       {{- range $index, $svc := $services.items -}}
+           {{- range $key, $val := $svc.metadata.labels -}}
+               {{- if and (eq $key "app") (contains "dashboard-svc-" $val) -}}
+{{- $svc.metadata.name | trim -}}
+               {{ end }}
+           {{- end }}
+       {{- end }}
+   {{- end }}
+{{- end }}
+
+{{- define "tyk-dev-portal.dashboardUrl" -}}
+{{ if ne .Values.overrideTykDashUrl "" }}
+    {{ .Values.overrideTykDashUrl }}
```

Comment:
Have you attempted to install the dev-portal with this option enabled? I have a suspicion that trailing spaces might be present, potentially causing indentation issues during the dev-portal deployment. However, I am not certain about this.

Reply:
Thank you for spotting it! Fixed in [b58506e](https://github.com/TykTechnologies/tyk-charts/pull/178/commits/b58506e9992104855da614e48f1c9464dad7d1ed)